build/bundle.js

```diff
@@ -20949,6 +20949,8 @@ var FCC_Global =
 
 	var _globalD3Tests = __webpack_require__(54);
 
+	var _alignmentTestsD = __webpack_require__(56);
```

Comment:
I believe it truncates modules names that end in a digit.   If you use `alignmentD3Test` it won't truncate.   I believe it does this because in situations where a 2nd variable is created in the bundle for example as _interopRequireDefault .   
```
var _jquery = __webpack_require__(1);
var _jquery2 = _interopRequireDefault(_jquery);
```
 The 2nd variable name will have incremented digit at end.   So they probably don't allow digits at end to avoid confusions there. 

If you keep the filname `alignmentTestD3.js`  , I don't think it would be a problem as the truncated name is just a variable name to reference the correct module/file.

But if change file name to `alignmentD3Test.js` could match `globalD3Test.js` naming style !